### PR TITLE
#2254 Update to CloudEvents 1.0, remove EventBroker

### LINKED ISSUE #2254: Switch to CloudEvents spec v1.0, remove eventbroker
Keptn currently utilizes CloudEvents in version 0.2. This issue upgrades the CloudEvents to version 1.0

**Tasks:**
- Upgrade https://github.com/cloudevents/sdk-go to version 2.2.0 in all services using CloudEvents
- cli: Switch to CloudEvents 1.0
- api-service, configuration-service, mongodb-datastore: adapt swagger models to be CloudEvents v1.0 conform
- distributor: **Directly** send CloudEvent to NATS or to the Keptn API
- gatekeeper-service, helm-service, jmeter-service, lighthouse-service,  openshift-route-service, remediation-service, shipyard-service: Send/receive CloudEvents v1.0 and send it to the distributor
- Remove `eventbroker`
- Update spec and documentation

**Definition of Done:**
- [ ] CLI upgraded
- [ ] api-service upgraded
- [ ] configuration-service upgraded
- [ ] mongodb-datastore upgraded
- [ ] distributor upgraded
- [ ] gatekeeper-service upgraded,
- [ ] helm-service upgraded
- [ ] jmeter-service upgraded
- [ ] lighthouse-service upgraded
- [ ] openshift-route-service upgraded
- [ ] remediation-service upgraded
- [ ] shipyard-service upgraded
- [ ] eventbroker removed
- [ ] All CloudEvents within Keptn are v1.0 conform
- [ ] Documentation is updated correspondingly

### PR DESCRIPTION
**Pull request by bacherfl**
Closes #2254 
This PR updates all Services within the repo to use the latest `keptn/go-utils` version which sends CLoudEvents using the CloudEvents v1.0 specification.
The functionality of the EventBroker has been integrated into the distributor. Execution plane services can now send outgoing events to their distributor sidecar running in the same pod, which will then forward the event to the NATS server. This way, execution plane services do not have to send events via the EventBroker anymore

### CODE REVIEW

File: api/utils/event.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+// SendEvent godoc
+func SendEvent(keptnContext, triggeredID, eventType string, data interface{}, l keptnutils.LoggerInterface) error {
+	source, _ := url.Parse("https://github.com/keptn/keptn/api")
+	ev := cloudevents.NewEvent()
+	ev.SetType(keptnevents.InternalProjectCreateEventType)
+	ev.SetID(uuid.New().String())
+	ev.SetTime(time.Now())
+	ev.SetSource(source.String())
```
Comment (agrimmer) [inline api/utils/event.go:35]: Am I right, that this method is used for forwarding events? If so, we should reuse the `source` of the incoming cloudevent.

File: api/go.mod [hunk 2]
```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/cloudevents/sdk-go v0.10.0
```
[the remark this answers is not in the document]
Comment (bacherfl) [reply api/go.mod]: good catch - this is not needed anymore

File: api/utils/event.go (hunk 1 above)
Comment (bacherfl) [reply api/utils/event.go]: you're right - I have modified the code to use the source that is passed to the function

**Review by agrimmer — APPROVED** [review]
lgtm

**Pull request merged**